package/MDAnalysis/core/topologyattrs.py

```diff
@@ -116,7 +119,7 @@ class Atomindices(TopologyAttr):
     """
     attrname = 'indices'
     singular = 'index'
-    level = 'atom'
+    levels = ['atom']
```

Comment:
So `Group.level` is now a `core.levels.Level` object that does fancy stuff, and this attribute `.levels` isn't that.  So maybe this should be `target_levels`